.github/workflows/images.yaml

```diff
@@ -0,0 +1,84 @@
+# NUC image creation and push to DockerHub
```

Comment:
Will also need to build and push `generic` in here.

Reply:
True